landoapi/transplants.py

```diff
@@ -763,34 +803,34 @@ def blocker_single_landing_repo(
     if not repo_phid:
         return "Landing path has no repository specified."
 
-    # Set the landing repo field on the `TransplantAssessmentState`.
-    landing_repo = transplant_state.landable_repos.get(repo_phid)
-    transplant_state.landing_assessment.landing_repo = landing_repo
+    # Set the landing repo field on the `LandingAssessmentState`.
+    landing_repo = stack_state.landable_repos.get(repo_phid)
+    stack_state.landing_assessment.landing_repo = landing_repo
 
 
-STACK_BLOCKER_LINTS = [
+STACK_BLOCKER_CHECKS = [
     # This check needs to be first.
     blocker_stack_landable,
     blocker_landing_already_requested,
     user_block_no_auth0_email,
     blocker_single_landing_repo,
 ]
 
-REVISION_BLOCKER_LINTS = [
+REVISION_BLOCKER_CHECKS = [
     user_block_scm_level,
     blocker_unsupported_repo,
     blocker_open_parents,
     blocker_closed_revisions,
     blocker_latest_diffs,
-    check_author_planned_changes,
-    check_diff_author_is_known,
-    check_uplift_approval,
-    check_revision_data_classification,
+    block_author_planned_changes,
+    block_diff_author_is_known,
+    block_uplift_approval,
+    block_revision_data_classification,
```

Comment:
Could be more consistent with the naming of other "blocker" checks (i.e., prefix with `blocker_` instead of `block_`, or rename everything to start with `block_`.